Gemfile

```diff
@@ -3,23 +3,27 @@
 
 source "https://rubygems.org"
 gem "logstash-core", "5.0.0.dev", :path => "./logstash-core"
-gem "logstash-core-event-java", "5.0.0.dev", :path => "./logstash-core-event-java"
+gem "logstash-core-event", "5.0.0.dev", :path => "./logstash-core-event"
 gem "logstash-core-plugin-api", "1.0.0", :path => "./logstash-core-plugin-api"
 gem "file-dependencies", "0.1.6"
 gem "ci_reporter_rspec", "1.0.0", :group => :development
 gem "simplecov", :group => :development
 gem "coveralls", :group => :development
-# Tins 1.7 requires the ruby 2.0 platform to install,
-# this gem is a dependency of term-ansi-color which is a dependency of coveralls.
-# 1.6 is the last supported version on jruby.
 gem "tins", "1.6", :group => :development
 gem "rspec", "~> 3.1.0", :group => :development
-gem "logstash-devutils", "~> 0.0.15", :group => :development
+gem "logstash-devutils", ">= 0", :path => "/Users/colin/dev/src/elasticsearch/logstash-devutils"
```

Comment:
yes, this is not a mergeable WIP and I added in the description that these must be addressed...